### ISSUE TITLE
Do not store exact addresses in Options

### ISSUE THREAD

**Issue by yfdyh000** [ilGur1132/Smart-HTTPS#6]
I do not know its role, the site servers should decide whether to provide HTTPS before got the address, and this quietly stores the user's privacy information, i.e. the visited urls, when first visit to a site.

**Comment by mrdokenny** [comment]
Yeah I thought about that too. I looked at the code and the exact url is used to reload the page when in detects an error so I tweaked it in 296a34ad4916c8dace688664a2c3ead2511e03b8 so that it doesn't store that data (only the domain). Also it should help with long addresses with breaks the table UI.

I also did some other stuff too so its not suitable for a PR but feel free to use the [fork](https://github.com/mrdokenny/Smart-HTTPS/releases).
